stable/oauth2-proxy/templates/deployment.yaml

```diff
@@ -149,6 +157,14 @@ spec:
 {{- end }}
         name: configaccesslist
 {{- end }}
+
+{{- if .Values.htpasswdFile.enabled }}
+      - configMap:
+          defaultMode: 420
+          name: {{ template "oauth2-proxy.fullname" . }}-htpasswd-file
```

Comment:
Wouldn't be better to map this as a k8s secret? 
Maybe add something like `htpasswd-file-existing-secret` as well, would be useful

Reply:
Hey @andrecastro, I implemented those changes. Nicer to have the file as a secret :+1: